pkg/validations/skipvalidations.go

```diff
@@ -0,0 +1,34 @@
+package validations
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ValidSkippableValidationsMap returns a map for all valid skippable validations as keys, defaulting values to false.
+// Defaulting to False means these validations won't be skipped unless set to True.
+func validSkippableValidationsMap(skippableValidations []string) map[string]bool {
+	validationsMap := make(map[string]bool, len(skippableValidations))
+
+	for i := range skippableValidations {
+		validationsMap[skippableValidations[i]] = false
+	}
+
+	return validationsMap
+}
+
+// ValidateSkippableValidation validates if provided validations are supported by EKSA to skip for upgrades.
+func ValidateSkippableValidation(skippedValidations []string, skippableValidations []string) (map[string]bool, error) {
+	svMap := validSkippableValidationsMap(skippableValidations)
+
+	for i := range skippedValidations {
+		validationName := skippedValidations[i]
+		_, ok := svMap[validationName]
+		if !ok {
+			return nil, fmt.Errorf("invalid validation name to be skipped. The supported validations that can be skipped using --skip-validations are %s", strings.Join(skippableValidations[:], ","))
```

Comment:
It would be nice if we could collect a list of these errors instead of failing on the first bad skipped validation, but I don't think it's worth blocking this PR on.

Reply:
Ah like say what all are invalid. Makes sense! 
So right now we have only 1 for upgrade and I will add one for create, so yes definitely something we can do as a follow.